lib/migration/feature/feature.go

```diff
@@ -28,6 +28,8 @@ const (
 	// create time is stored in nanoseconds (previously seconds) in the portlayer.
 	ContainerCreateTimestampVersion
 
+	VMFolderSupportVersion
```

Comment:
Minor: s/VMFolderSupportVersion/VCHFolderSupportVersion perhaps?

A short comment to describe this field for posterity would be nice to have.

Reply:
Yeah I wondered whether it was a catch-all (VM) or exclusive to the VCH. I'll clarify.